README.md

```diff
@@ -390,6 +391,22 @@ Schema::create('test', function (Blueprint $table) {
 });
 ```
 
+### Full-text search using FULLTEXT indexes
+
+SingleStoreDB supports full-text search across text columns in a columnstore table using the `FULLTEXT` index type.
+Keep in mind that `FULLTEXT` is not supported when using the `utf8mb4` collation, the table needs to be set to the collation `utf8_unicode_ci`. If you try to add a `FULLTEXT` index to a table with the `utf8mb4` collation an exception will be thrown.
```

Comment:
I think only the column needs the utf8 collation not the entire table. Can you check?

Reply:
That's correct!

Reply:
I made a few tweaks to the wording to make it simpler, and adjusted the tests

Reply:
I took the opportunity to add a test about that exception https://github.com/singlestore-labs/singlestoredb-laravel-driver/pull/30/commits/b5b12dd1585eb2ac151dba00738660c422eb512c

If support is added in the future then we'll have a test to remind us to make the required changes